scripts/preconversion_assessment_script.py

```diff
@@ -11,6 +12,23 @@
     from urllib2 import urlopen
```

Comment:
Only remaining question is for python3 usage, just for fun I tried to run it with python 3.6, and the script fails because it looks like urlopen returns bytes instead of string in that version, then I get error that write argument must be string not bytes (in `setup_convert2rhel`) 

This is the output:

```
Checking if required files are present on the system.
Checking for file /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
Checking for file /etc/rhsm/ca/redhat-uep.pem
Checking for file /etc/yum.repos.d/convert2rhel.repo
Downloading required files.
Writing file to destination: '/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release'
Cleaning up modifications to the system.
Removing the file '/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release' as it was previously downloaded.
File '/etc/rhsm/ca/redhat-uep.pem' was present on the system before the execution. Skipping the removal.
File '/etc/yum.repos.d/convert2rhel.repo' was present on the system before the execution. Skipping the removal.
Analysis was successfully executed.
### JSON START ###
{
    "status": "ERROR",
    "message": "",
    "report": "write() argument must be str, not bytes",
    "report_json": null
}
### JSON END ###
```

We could also make the print with `Analysis was successfully executed.` only if it actually run successfully (probably not in finaly, or change it that execution just ended?), maybe whole part that is done in finally could be visually divided from the execution prints - just to clearly see when it went bad/when it stopped

Reply:
LGTM! :+1: 

Just a note: talked with @r0x0d and we decided to drop python3 for now so the error in case of wrong version is clear, output was updated as suggested in previous comment